toolkit/featureforms/src/main/java/com/arcgismaps/toolkit/featureforms/internal/navigation/FeatureFormNavHost.kt

```diff
@@ -101,7 +101,7 @@ internal fun FeatureFormNavHost(
         composable<NavigationRoute.UNAssociationsView> { backStackEntry ->
             val route = backStackEntry.toRoute<NavigationRoute.UNAssociationsView>()
             val formData = remember(backStackEntry) { state.getActiveFormStateData() }
-            UNAssociationsScreen(
+            UNAssociationsGroupScreen(
```

Comment:
Should it be `Groups`?
```suggestion
            UNAssociationsGroupsScreen(
```

Reply:
Since it shows a single group, I've updated it to `UNAssociationGroupResultScreen`.